### PR TITLE
Create mrstickypiston.json

### PR DESCRIPTION
**Pull request by MrStickyPiston**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

screenshot of my amazing material design website

### CODE REVIEW

File: domains/mrstickypiston.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "MrStickyPiston",
+        "email": "email@gmail.com"
```
Comment (Stef-00012) [inline domains/mrstickypiston.json:4]: You should put your own mail

**Review by Stef-DP — DISMISSED** [review]

**Comment by MrStickyPiston** [comment]
Fixed it

**Comment by TobezDev** [comment]
Can I just add here that the site is incomplete.
Most of the text content is just a placeholder which can be generated from sites such as https://loremipsum.io.

[Suggestion: add tag: `reason: incomplete`]

**Comment by orxngc** [comment]
@tobezdev Yah a lot of it is lorem ipsum, but all you really need to be considered complete is your name and mayhaps contact info, both of which are present.

**Pull request merged**